modin/tests/pandas/dataframe/test_join_sort.py

```diff
@@ -80,20 +80,20 @@ def test_combine(data):
     "test_data, test_data2",
     [
         (
-            np.random.uniform(0, 100, size=(2**6, 2**6)),
-            np.random.uniform(0, 100, size=(2**7, 2**6)),
+            np.random.randint(0, 100, size=(64, 64)),
+            np.random.uniform(0, 100, size=(128, 64)),
```

Comment:
```suggestion
            np.random.randint(0, 100, size=(128, 64)),
```
Should we change this too?

Reply:
Goog catch!